manifests/Nesbox/TIC-80/0.70.6.yaml

```
@@ -0,0 +1,18 @@
+Id: Nesbox.TIC-80
+Version: 0.70.6
+Name: TIC-80
+Publisher: Nesbox
+License: MIT
+LicenseUrl: https://github.com/nesbox/TIC-80/blob/v0.70.6/LICENSE
+AppMoniker: tic80
+Tags: retro,8bit,gamedev,fantasy computer
+Description: TIC-80 is a fantasy computer for making, playing and sharing tiny games.
+Homepage: https://tic80.com/
+Installers:
+  - Arch: x86
+    Url: https://github.com/nesbox/TIC-80/releases/download/v0.70.6/tic80_0.70.6.exe
+    Sha256: 921AE257A6187E52862194FA5AEA1EFFF9341690B0CEC92AB9015F71D50CDBBC
+    InstallerType: nullsoft
+    Switches: 
+      Silent: /S
+      SilentWithProgress: /S
```

```suggestion
    InstallerType: nullsoft
```

those switches are already included in the nullsoft type